test/e2e/tests/relayer.go

```diff
@@ -66,9 +66,29 @@ func (r *RelayerTestSuite) TestRelayer(ctx context.Context) {
 		r.Require().Equal(1, seq)
 	})
 
-	r.T.Run("test crash and recover", func(t *testing.T) {
-		duration, err := r.CrashAndRecover(ctx)
+	r.T.Run("test packet flow", func(t *testing.T) {
+		chainA, chainB := r.GetChains()
+		r.PacketFlow(ctx, chainA, chainB, "test")
+	})
+
+	r.T.Run("crash nodes", func(t *testing.T) {
+		chainA, chainB := r.GetChains()
+		r.Require().NoError(r.CrashNode(ctx, chainA))
+		r.Require().NoError(r.CrashNode(ctx, chainB))
+	})
+
+	r.T.Run("test crash and recover relay", func(t *testing.T) {
```

Comment:
To make this test more realisitic i would want packets to be sent while the relayer is down,
Prefarably, sent, acknowledged and timed out and would like to see it recover

Reply:
Improved.

Reply:
Not sure i understand? 
isnt the flow now just
send packet and relay 
Close relay
start Relay
ExecuteCall

Which means the relay does not anything after recovery?

Reply:
- Crash Both nodes, chainA and B
- Send Packets to both nodes
- Crash Relay
- Recover both chain's node
- Recover Relay
- Get sent packets before crash
- Ping Pong cross chain test

Reply:
Aah okey so relay never get notified sice nodes are down when you publish the inital package?

Reply:
I am supposed to crash destination chain instead of both.

Reply:
Anton I have pushed the changes that may satisfies you're talking about?